src/shellintrospect.h

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright © 2019 Red Hat, Inc
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2 of the License, or (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library. If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+#pragma once
+
+#include <glib-object.h>
+
+typedef struct _Window Window;
+
+const char * window_get_app_id (Window *window);
+
+const char * window_get_title (Window *window);
+
+const guint64 window_get_id (Window *window);
+
+GList * get_available_windows ();
+
```

Comment:
Trailing empty line

Reply:
Opps, I added on purpose. Seems none of the other files do it. Done!